test/demo/raiseIntent/intentResolution.inttest.ts

```diff
@@ -0,0 +1,192 @@
+import 'jest';
```

Comment:
Does this need to be its own file? What's the logic for what goes in here, and what goes in `withTarget` or `withoutTarget`